gnovm/pkg/test/imports.go

```diff
@@ -228,9 +228,11 @@ func loadStdlib(rootDir, pkgPath string, store gno.Store, stdout io.Writer, prep
 		// Normal stdlib path.
 		filepath.Join(rootDir, "gnovm", "stdlibs", pkgPath),
 	}
+	var mPkgType gno.MemPackageType
 	if testing {
 		// Override path. Definitions here override the previous if duplicate.
 		dirs = append(dirs, filepath.Join(rootDir, "gnovm", "tests", "stdlibs", pkgPath))
+		mPkgType = gno.MPStdlibTest
 	}
```

Comment:
Yet to test more; we might need to bring in different MPStdlibXXX for different types of testing.